src/View/ProgressWindow.py

```diff
@@ -59,15 +60,58 @@ def update_progress(self, progress_update):
-                                                               "like to calculate DVHs? (This may take up to several "
-                                                               "minutes on some systems.)",
-                                      QMessageBox.Yes | QMessageBox.No)
-
-        if choice == QMessageBox.Yes:
-            self.signal_advise_calc_dvh.emit(True)
+        """
+            Windows displays buttons in a different order from Linux. A check for
+            platform is performed to ensure consistency of button positioning across
+            platforms.
+        """
+        if platform.system() == "Linux":
+            choice = QMessageBox.question(self, "Calculate DVHs?", "RTSTRUCT and RTDOSE datasets identified. Would you "
+                                                                   "like to calculate DVHs? (This may take up to "
+                                                                   "several minutes on some systems.)",
+                                                QMessageBox.Yes | QMessageBox.No)
+
+            if choice == QMessageBox.Yes:
+                self.signal_advise_calc_dvh.emit(True)
+            else:
+                self.signal_advise_calc_dvh.emit(False)
         else:
-            self.signal_advise_calc_dvh.emit(False)
+            stylesheet_path = ""
+
+            # Select appropriate style sheet
+            if platform.system() == 'Darwin':
+                stylesheet_path = "res/stylesheet.qss"
+            else:
+                stylesheet_path = "res/stylesheet-win-linux.qss"
+
+            # Create a message box and add attributes
+            mb = QMessageBox()
+            mb.setIcon(QMessageBox.Question)
+            mb.setWindowTitle("Calculate DVHs?")
+            mb.setText("RTSTRUCT and RTDOSE datasets identified. Would you "
+                       "like to calculate DVHs? (This may take up to several "
+                       "minutes on some systems.)")
+            button_no = QtWidgets.QPushButton("No")
+            button_yes = QtWidgets.QPushButton("Yes")
+
+            """ We want the buttons 'No' & 'Yes' to be displayed in that exact order. QMessageBox displays buttons in
+                respect to their assigned roles. (0 first, then 0 and so on) 'AcceptRole' is 0 and 'RejectRole' is 1 
+                thus by counterintuitively assigning 'No' to 'AcceptRole' and 'Yes' to 'RejectRole' the buttons are 
+                positioned as desired.
+            """
+            mb.addButton(button_no, QtWidgets.QMessageBox.AcceptRole)
+            mb.addButton(button_yes, QtWidgets.QMessageBox.RejectRole)
+
+            # Apply stylesheet to the message box and add icon to the window
+            mb.setStyleSheet(open(stylesheet_path).read())
+            mb.setWindowIcon(QtGui.QIcon(resource_path("res/images/btn-icons/onkodicom_icon.png")))
```

Comment:
Is this expression of path platform independent?  If not, suggest using pathlib.

Reply:
I believe it was, it's been updated to use pathlib regardless.